tests/xtro-sharpie/xtro-sanity/Sanitizer.cs

```diff
@@ -212,12 +221,21 @@ public static int Main (string [] args)
 					continue;
 				var rawfile = Path.ChangeExtension (file, ".raw");
 				var raws = new List<string> (File.ReadAllLines (rawfile));
-				foreach (var entry in File.ReadAllLines (file)) {
+				var failures = new List<string> ();
+				var lines = File.ReadAllLines (file);
```

Comment:
We used to get issues when doing ReadAllLines in large files in xharness results. Can we use a reader instead?

Reply:
They are not the big - but an easier change is to use `foreach (var line in File.ReadLines(file))` which will read line-by-line (in memory)